package/ci/appveyor-desktop.bat

```diff
@@ -142,6 +142,7 @@ cmake .. ^
     -DWITH_STBVORBISAUDIOIMPORTER=ON ^
     -DWITH_STLIMPORTER=ON ^
     -DWITH_TINYGLTFIMPORTER=ON ^
+    -DWITH_WEBPIMPORTER=ON ^
```

Comment:
```suggestion
    -DWITH_WEBPIMPORTER=OFF ^
```

I saw you temporarily added a download on Windows via vcpkg. Maybe that would have worked, but often it ends up being a real nightmare, working on some compilers and some not, and breaking every second Monday. I'll try that later, maybe. For now, to get the CI green, let's just not bother.